docs/excel/custom-functions-json.md

```diff
@@ -165,17 +178,35 @@ The `functions` property is an array of custom function objects. The following t
 | `parameters`  | array     | Yes      | Array that defines the input parameters for the function. See [parameters](#parameters) for details.                                                                             |
 | `result`      | object    | Yes      | Object that defines the type of information that is returned by the function. See [result](#result) for details.                                                                 |
 
+### enums
+
+The `enums` property is an array of [enum](https://www.typescriptlang.org/docs/handbook/enums.html) constants. The following table lists the properties of each constant.
```

Comment:
In the JSON it looks like it is metadata describing enums. It's not an actual array of enums in the JSON.